mods/cnc/rules/vehicles.yaml

```diff
@@ -512,6 +518,9 @@ HTNK:
 		Delay: 10
 		StartIfBelow: 50
 		DamageCooldown: 200
+	Explodes:
+		Weapon: ArtilleryShell
```

Comment:
Shouldn't we be adding a new `UnitExplodeArtillery` for this?

Reply:
I tested it and it works, but keep in mind that Mammoth Tanks must also have the same effects. So is it okay to add `UnitExplodeArtillery` to them too? I already added a new `UnitExplodeBig` with different effects.

Reply:
Ah, i see. We should rename this to something more clearly related to the animation/sound then.

Reply:
If I'm not mistaken only Nod's Artillery, GDI' MLRS (which are called MSAM in the vehicles.yaml btw) and Mammoth Tanks share the same explosion/sound effects.